servant-foreign/src/Servant/Foreign/Internal.hs

```diff
@@ -130,13 +130,14 @@ data Req f = Req
   , _reqBody       :: Maybe f
   , _reqReturnType :: Maybe f
   , _reqFuncName   :: FunctionName
+  , _reqBodyIsJSON :: Bool
```

Comment:
I'd rather have

```haskell
reqBodyContentType :: ReqBodyContentType

data ReqBodyContentType = RBCTJSON | RBCTRaw
```

(better names are welcome). I'd like `servant-foreign` to support other "envelope" formats (cbor, edn, ... - those aren't easy to use from JavaScript, but there are other client languages!)

Reply:
I did that too.